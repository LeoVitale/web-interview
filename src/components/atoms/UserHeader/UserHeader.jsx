import React from 'react';
import styles from './UserHeader.module.scss';

const UserHeader = ({ title, subheader }) => {
  const { userHeader } = styles;
  return (
    <div className={userHeader}>
      {title && <span className={styles.title}>{title}</span>}
      {subheader && <span className={styles.subheader}>{subheader}</span>}
    </div>
  )
}

export default UserHeader
