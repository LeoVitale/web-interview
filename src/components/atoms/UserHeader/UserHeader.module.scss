@import 'styles/_constants.scss';

.userHeader {
  span {
    display: block;
    &.title {
      font-size: 1.2em;
    }
    &.subheader {
      font-size: 1em;
      color: $transparent-grey;
    }
  }
}
