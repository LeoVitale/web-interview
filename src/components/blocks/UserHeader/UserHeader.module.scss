@import 'styles/_constants.scss';

.userHeader {
  span {
    display: block;
    &.title {
      font-size: 1.6em;
    }
    &.subheader {
      font-size: 1.4em;
      color: $transparent-grey;
    }
  }
}
