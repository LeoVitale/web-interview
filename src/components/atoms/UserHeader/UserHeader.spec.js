import React from 'react'
import { mount } from 'enzyme'
import UserHeader from './UserHeader.jsx'

describe('<UserHeader />', () => {
  const userHeader = mount(<UserHeader />)

  it('should render UserHeader as expected', () => {
    expect(userHeader).toMatchSnapshot()
  })
})
