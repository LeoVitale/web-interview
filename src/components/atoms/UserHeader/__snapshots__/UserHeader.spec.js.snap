// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserHeader /> should render UserHeader as expected 1`] = `
<UserHeader>
  <div />
</UserHeader>
`;
